public/static/app/perf-bb/PerfBBRejectsCtrl.js

```diff
@@ -1,34 +1,24 @@
 mciModule.controller('PerfBBRejectsCtrl', function (
-  $scope, $window, EvgUiGridUtil, EvgUtil, FORMAT, MDBQueryAdaptor, uiGridConstants,
-  STITCH_CONFIG, Stitch, Settings, $timeout, $compile, $log, WhitelistDataService, $q, RejectState, $mdDialog
+  $scope, $window, EvgUiGridUtil, EvgUtil, FORMAT, MDBQueryAdaptor, uiGridConstants, confirmDialogFactory,
+  STITCH_CONFIG, Stitch, Settings, $timeout, $compile, $log, WhitelistDataService, $q, RejectState, Lock
 ) {
   // Perf Rejects View-Model.
   const vm = this;
   const project = window.project;
   const LIMIT = 2000;
 
-  vm.locked = false;
+  vm.lock = new Lock();
 
-  $scope.lock = () => vm.locked = true;
-  $scope.unlock = () => vm.locked = false;
-  $scope.locked = () => vm.locked;
-
-  const  addDisabled = () => $scope.locked() || vm.selection.length === 0 || _.all(vm.selection, (doc)=> doc.whitelisted);
-  const  removeDisabled = ()  => $scope.locked() || vm.selection.length === 0 || _.all(vm.selection, (doc)=> !doc.whitelisted);
-
-  function confirmMarkAction(items, action) {
-    return $mdDialog.show(
-      $mdDialog.confirm()
-        .ok('Ok')
-        .cancel('Cancel')
-        .title('Confirm')
-        .textContent(action + ' ' + items.length + ' item(s)?')
-    );
-  }
+  $scope.lock = vm.lock;
+  const confirmAdd = confirmDialogFactory('Add whitelisting for');
+  const confirmRemove = confirmDialogFactory('Remove whitelisting for');
+  const  addDisabled = () => $scope.lock.locked || vm.selection.length === 0 || _.all(vm.selection, (doc)=> doc.whitelisted);
```

Comment:
[nit] The spacing of the anonymous functions on this line and the following line is inconsistent. Can they all the update to have a single space between '()' and '=>'?

Reply:
Looks like there is just some bad white space between the 2 lines. There should be a single space between the round brackets and the arrow,